### ISSUE TITLE
Uptime on Status Page

### ISSUE THREAD

**Issue by markus101** [Sonarr/Sonarr#1056]
We'd need to track this (in memory) and show it on the page.

**Comment by tidusjar** [comment]
Any preference where on the status page?

**Comment by markus101** [comment]
In the `About` section below Startup Directory makes sense I think.

**Comment by Taloth** [comment]
Can I ask why we would want to clutter the UI with that?

**Comment by tidusjar** [comment]
Sonarr uptime or system uptime?

**Comment by markus101** [comment]
Sonarr uptime.

@Taloth I don't know the user's exact reasoning for it, here is the original request: https://forums.sonarr.tv/t/show-uptime-on-status-page/8948 This is a proposal because I wasn't sure the value, but didn't want to be the sole decision maker.

**Comment by Taloth** [comment]
I do think it clutters coz it's not that useful. There is other stuff (indexer stats) that would be more useful.
Nr of recent releases scanned recently, nr of recent grabs, downloads, sizes etc. Add that all up and you already have enough stuff for that page.

It's not that I'm against adding it. Also haven't checked the new UI to see if it would fit better there.

**Comment by vawen** [comment]
That stat section sound pretty nice. Which indexer grabbed more releases, time between grabbed and downloaded per indexer (prolly torrent only), so much useful info....

**Comment by Taloth** [comment]
Yup, those stats can be retrieved fairly easily from the db. Overal it should be no more than a few hours work to get this whole feature done (excluding code reviews and rewrites :wink:). Sadly I have a gazilion PRs to handle first.

**Comment by thezoggy** [comment]
ties in with https://github.com/Sonarr/Sonarr/issues/323
